assets/src/components/detourListPage.tsx

```diff
@@ -0,0 +1,120 @@
+import React from "react"
+import { Table } from "react-bootstrap"
+import { RoutePill } from "./routePill"
+import { useCurrentTimeSeconds } from "../hooks/useCurrentTime"
+import { timeAgoLabel } from "../util/dateTime"
+import useScreenSize from "../hooks/useScreenSize"
+
+export const DetourListPage = () => {
+  const fakeData = [
+    {
+      number: 45,
+      direction: "Outbound",
+      name: "Franklin Park via Ruggles Station",
+      intersection: "John F. Kennedy St & Memorial Drive",
+      activeSince: 1722372950,
+    },
+    {
+      number: 83,
+      direction: "Inbound",
+      name: "Central Square",
+      intersection: "Pearl Street & Clearwell Ave",
+      activeSince: 1722361948,
+    },
+    {
+      number: 83,
+      direction: "Outbound",
+      name: "Rindge Ave",
+      intersection: "Pearl Street & Clearwell Ave",
+      activeSince: 1721361948,
+    },
+    {
+      number: 45,
+      direction: "Outbound",
+      name: "Franklin Park via Ruggles Station",
+      intersection: "John F. Kennedy St & Memorial Drive",
+      activeSince: 1722372950,
+    },
+    {
+      number: 83,
+      direction: "Inbound",
+      name: "Central Square",
+      intersection: "Pearl Street & Clearwell Ave",
+      activeSince: 1722361948,
+    },
+    {
+      number: 83,
+      direction: "Outbound",
+      name: "Rindge Ave",
+      intersection: "Pearl Street & Clearwell Ave",
+      activeSince: 1721361948,
+    },
+    {
+      number: 45,
+      direction: "Outbound",
+      name: "Franklin Park via Ruggles Station",
+      intersection: "John F. Kennedy St & Memorial Drive",
+      activeSince: 1722372950,
+    },
+    {
+      number: 83,
+      direction: "Inbound",
+      name: "Central Square",
+      intersection: "Pearl Street & Clearwell Ave",
+      activeSince: 1722361948,
+    },
+    {
+      number: 83,
+      direction: "Outbound",
+      name: "Rindge Ave",
+      intersection: "Pearl Street & Clearwell Ave",
+      activeSince: 1721361948,
+    },
+  ]
+
+  const epochNowInSeconds = useCurrentTimeSeconds()
+  const screenSize = useScreenSize()
+
+  return (
+    <>
+      <Table hover className="c-detour-table">
```

Comment:
How would you feel about extracting this to a separate component, like `<DetoursTable data={...}/>` or something like that?

I see two benefits from that:
1. Storybook
2. As we iterate on styles and bootstraps and mobile layouts etc, this page doesn't have to change at all - only the component (whose changes would then show up in Storybook)

Reply:
Fwiw, I think it's worth waiting on making this it's own component.

Reply:
I did like moving to its own component, I think that's fine and went ahead and did that. (Kayla, feel free to still weigh in on why it's worth waiting.) But didn't add to Storybook yet, because I see that as "visual change control" and think that can/should come in once the table is properly formatted. Thoughts?

Reply:
> Kayla, feel free to still weigh in on why it's worth waiting

I think we tend to "over-componentize" and it prevents us from using composition to our advantage and forces us into managing props and prop drilling more often. I think react makes it seem like you're supposed to make a component out of everything but I think acting too soon leaves you with bad abstractions and bad data structures.

That said, now that it's done, I wouldn't revert.

Reply:
Hmm I can see that. In this case, this table will be repeated 3 times on the page with slight variation, so I saw it as an especially apt candidate for a component. (Initially, I wasn't sure if I should do it now or later once we populate the tables, but figured I might as well after Josh mentioned it, too)

Reply:
I agree with everything both of you said, and still think that this in particular deserves to be a component.

Relatedly...
> acting too soon leaves you with bad abstractions and bad data structures.

I think we have a few bad abstractions and bad data structures lying around (at least in our detours code), and I definitely support refactoring them to be less bad.

Reply:
As far as Storybook is concerned, I think it's still valuable to have a source of truth of *What Is*, and I actually think that it's even more valuable right now because of the contrast between *What Is* and *What Should Be*.

Plus, (I dunno maybe this is just me) it's extremely satisfying to see the Chromatic diff when we improve the design of a component.

Happy to let Storybook be a follow-up though. No need for it to be in this PR.